packages/core/src/schema/models/publishedDocuments.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  bigint,
+  bigserial,
+  boolean,
+  index,
+  uniqueIndex,
+  uuid,
+} from 'drizzle-orm/pg-core'
+
+import { latitudeSchema } from '../db-schema'
+import { timestamps } from '../schemaHelpers'
+import { projects } from './projects'
+import { workspaces } from './workspaces'
+
+export const publishedDocuments = latitudeSchema.table(
+  'published_documents',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    uuid: uuid('uuid').notNull().unique(),
+    documentUuid: uuid('document_uuid').notNull(),
+    workspaceId: bigint('workspace_id', { mode: 'number' })
+      .notNull()
+      .references(() => workspaces.id, { onDelete: 'cascade' }),
+    projectId: bigint('project_id', { mode: 'number' })
+      .notNull()
+      .references(() => projects.id, { onDelete: 'cascade' }),
+    allowFollowConversation: boolean('allow_follow_conversation').notNull().default(false),
+    ...timestamps(),
+  },
+  (table) => ({
+    projectWorkspaceIdx: index('workspace_idx').on(table.workspaceId),
+    uniqueProjectDocumentUuid: uniqueIndex(
+      'unique_project_document_uuid_idx',
+    ).on(table.projectId, table.documentUuid),
+  }),
```

Comment:
My idea here is to look these public documents by `uuid` and then pick 

`workspaceId`, `projectId` and `documentUuid` and use our methods
```javascript
const documentsScope = new DocumentVersionsRepository(ctx.workspace.id)
const commitUuid = // Get live version for this project
const document = await documentsScope
  .getDocumentAtCommit({
    projectId,
    commitUuid,
    documentUuid,
  })
  .then((r) => r.unwrap())
```

What do you think?